src/MessagingInput/MessagingInput.tsx

```diff
@@ -150,8 +157,14 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
               }
             }}
             placeholder={placeholder}
-            onBlur={onBlur}
-            onFocus={onFocus}
+            onBlur={() => {
+              setIsInputActive(false);
+              onBlur();
+            }}
+            onFocus={() => {
+              setIsInputActive(true);
+              onFocus();
+            }}
```

Comment:
We use this state to tweak the styles of the reply area when the input area is focused. There unfortunately wasn't a good way to use CSS alone to target the reply area when the input area is focused. The sibling `+` selector almost does the trick but only works for capturing the second element of two adjacent elements, not the first, which is what we need here (since the reply area comes before the input area in the DOM).